upgrading/topics/keycloak/changes.adoc

```diff
@@ -1,5 +1,41 @@
 == Migration Changes
 
+=== Migrating to 13.0.0
+
+==== Upgrade to Wildfly 23
+
+The {project_name} server was upgraded to use Wildfly 23 as the underlying container. This does not directly involve any
+specific {project_name} server functionality, but a few changes related to the migration, which are worth mentioning.
```

Comment:
Change " but a few changes related to the migration, which are worth mentioning"
TO
"however, note these changes related to migration"

Reply:
Thanks, Andy. Fixed all ten instances.